paths/reset/fc-scenarios.md

```diff
@@ -0,0 +1,66 @@
+
+  Unlike our other courses, this class isn't linear which means before you can begin working on any scenario, you are going to need to do a little bit of setup to get a 'testing environment' ready.
+
+   1. Create a repository on GitHub.com and `clone` it to your desktop.
+
+       With a remote repository created, we need to populate our repository with some files and commits.
+
+   1. Before creating new files, you should probably create a new branch, maybe call it something like ```test```.
+
+  With a branch created, its time to create those files, thankfully we have some handy scripts that will automatically generate files and commits (with commit messages!) that will make your life a little easier.
+
+   - **Bash:** ```for d in {1..6}; do touch file$d.md; git add file$d.md; git commit -m "adding file $d"; done```
+   - **PowerShell:** ```for ($d=1; $d -le 6;$d++) { touch file$d.md; git add file$d.md; git commit -m "adding file$d.md";}```
+
+  If you created a repository on GitHub.com, cloned it, created a branch, and created files and commits, you are ready to rock and roll! Sorry about the GIANT block of text to get setup :cry:.
+
+  ## Scenarios
+  - [Committed Something on Accident](/on-demand/reset/fc-05)
+
+    During your workflow, you might have gotten excited and made a commit before you had actually finished your thought. Maybe you made the commit without adding all of the related files. Perhaps your commit included changes that were unrelated. In either case, we can fix that, just follow along and `git` out of trouble.
+
+  - [Commit to Wrong Branch](/on-demand/reset/fc-04)
+
+    So you got excited about making some changes and accidentally committed your changes to the wrong branch? Happens **all** the time. This scenario walks you through the steps required to successfully remove those commits and commit them to the right branch, even if you already pushed them up!
+
+  - [Commit Message Sucks](/on-demand/reset/fc-03)
+
+    'Edited stuff' might have seemed like a decent commit message at 3am, but in hindsight, you might realize you want to provide more context. Covering the different commands you can use to `revert` the error of your commit message-ways, this scenario identifies how to get more descriptive.
+
+  - [Commit Broke EVERYTHING](/on-demand/reset/fc-06)
+
+    The sky is **not** falling down chicken little, we can make everything right-as-rain, and this scenario shows you how to fix that last commit.
+
+  - [Too Many Commits!](/on-demand/reset/fc-02)
+
+    While you were making changes you might have created a bunch of tiny commits to keep track of the changes you were making, but when it comes time to actually push your changes back to your `remote`, you want to group your commits together and prevent your commit history from being inundated with the 30 commits you just made. This scenario guides you through `squashing` those commits and creating more condensed commits.   
+
+show-me-how:
+tell-me-why: |
```

Comment:
Was this ❓ for the errant 'the' at the bottom of the page, or something else? Or is the content on line 61 just confusing?